ocm_python_wrapper/cluster.py

```diff
@@ -372,14 +372,15 @@ def uninstall_addon(self, wait=True, wait_timeout=TIMEOUT_30MIN):
     @staticmethod
     def update_rhoam_cluster_storage_config():
         def _wait_for_rhmi_resource():
-            for sample in TimeoutSampler(
+            _rhmi = RHMI(name="rhoam", namespace="redhat-rhoam-operator")
```

Comment:
Patching now works, but this line fails with non-existence (trying to reach too early before resource created):
`ocp_resources.resource WARNING Couldn't find RHMI in integreatly.org api group`
`*** NotImplementedError: Couldn't find RHMI in integreatly.org api group`